.github/workflows/process-load-test-result.yml

```diff
@@ -52,14 +54,19 @@ jobs:
       - name: Merge PR
         if: ${{ steps.processResults.outputs.hasPassed }}
         run: |
-            status="NOT COMPLETED"
-            while [ ! -z "$status" ]
-            do
-              sleep 20
-              status=$(gh pr status --jq '.currentBranch .statusCheckRollup[] | select(.status!="COMPLETED") | select(.context!="license/cla")' --json statusCheckRollup)
-            done
+          checkCount="0"
+          while [ "$checkCount" != "3" ]
+          do
             sleep 20
-            gh pr merge --merge --delete-branch
+            checkCount=$(gh pr status --jq '[.currentBranch .statusCheckRollup[] | select((.conclusion=="SUCCESS") and ((.name=="Build on Ubuntu") or (.name=="Build on Windows") or (.name=="codecov/project")))] | length' --json statusCheckRollup)
+            if [[ "$failedCount" != "0" ]]
```

Comment:
need to add `failedCount=$(gh pr status --jq '[.currentBranch .statusCheckRollup[] | select((.conclusion=="FAILURE") and ((.name=="Build on Ubuntu") or (.name=="Build on Windows") or (.name=="codecov/project")))]|length' --json statusCheckRollup)` as well

Reply:
Fixed with [235c7646e2dfd2ef7d2e1528dc48d89ba322b882](https://github.com/ballerina-platform/module-ballerina-http/pull/896/commits/235c7646e2dfd2ef7d2e1528dc48d89ba322b882)